plugins/clipboard/plugin.js

```diff
@@ -2825,6 +2825,12 @@
 
 					fallbackDataTransfer._isCustomMimeTypeSupported = false;
 
+					// It looks like after our custom mime type test Edge 17 is denying access on nativeDataTransfer (#2169).
+					// Upstream issue: https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/18089287/
+					if ( CKEDITOR.env.edge && CKEDITOR.env.version > 16 ) {
```

Comment:
`CKEDITOR.env.version` is a float, not integer. In this case this conditional is targeting also 16.x Edge.